cmd/soroban-rpc/internal/methods/get_transaction.go

```diff
@@ -29,8 +29,6 @@ const (
 
 // GetTransactionResponse is the response for the Soroban-RPC getTransaction() endpoint
 type GetTransactionResponse struct {
-	// Status is one of: TransactionSuccess, TransactionNotFound, or TransactionFailed.
-	Status string `json:"status"`
```

Comment:
We could still keep the `Status` field to show what actually happened with Txn. Any strong motive to remove this field? or is it rarely used by users?

Reply:
Well it's still part of `TransactionInfo`. It can take on one additional value (`TransactionNotFound`) for this endpoint, but since it's a `string` regardless this doesn't impact the code.